src/routes/articles/get-article.e2e.test.js

```diff
@@ -0,0 +1,82 @@
+const {expect} = require('chai')
+const {fixtures} = require('../../../test/utils')
+
+
+describe('GET /Articles/:article.id', () => {
+    
+	let user
+	let accessToken
+
+	before(async ()=>{
+		 user = await fixtures.insertUser() 
+		const body = { id: user.id, email: user.email }
+		accessToken = fixtures.generateAccessToken(
+			{user : body}
+		)
+	})
+
+	describe('Failure', () => {
+
+		it('should return 400 if article.id is invalid', async () => {
+			const expectedError = {
+				"status" : "failed",
+				"error": {
+			  "message": "id must be a number"
+				}
+		  }
+			return	fixtures.api()
+				.get(`/api/v1/articles/sdsdsd`)
+				.set('Authorization', `Bearer ${accessToken}`)
+				.expect(400, expectedError)
+
+		})
+		
+	})
+
+	describe('Success', () => {
+
+		let article
+
+		before(async ()=>{
+			article = await fixtures.insertPost({
+				userId : user.id , 
+				type : 'article'
+			})
+		})
+
+		it('should return 200', async () => 
+
+			fixtures.api()
+				.get(`/api/v1/articles/${article.id}`)
+				.set('Authorization', `Bearer ${accessToken}`)
+				.expect(200)
+		)
+
+		it('should return the right response', async () =>
```

Comment:
Since the comment is part of the specification of the endpoint so we would also like to know that the comment is being returned as it should in the data 

So let's kindly insert a comment as well through the utils and expect it as part of the data

Reply:
i have made the change and pushed to github